### ISSUE TITLE
Error when running ./quickstart.sh

### ISSUE THREAD

**Issue by ablwr** [bavc/signalserver#257]
Error reads as such:

```
WARNING: The FILES_VOLUME_PATH variable is not set. Defaulting to a blank string.
Creating volume "signalserver_dbdata" with default driver
Creating volume "signalserver_files" with default driver
ERROR: create .: volume name is too short, names should be at least two alphanumeric characters
``` 

I've tried making and also associating the /data and /files directories but that doesn't seem to help. Also, I opened up the permissions to /files completely, 777.

**Comment by yayoiukai** [comment]
okay it is strange. I will look.

**Comment by yayoiukai** [comment]
Did you set FILES_VOLUME_PATH=/files? It is written in README. Either you can set an environmental variable, or you can create the .env file. Please let me know if it doesn't work still.

**Comment by ablwr** [comment]
Yup I tried it with that and without it, and with open permissions. I pulled the latest and am trying again!

**Comment by ablwr** [comment]
No, I still get the same error. 

I ran everything as the README suggests and also tried running quickstart by un-commenting out the first couple of lines, but still the same error!

**Comment by yayoiukai** [comment]
Hmm, that's a problem. Did you update docker? I am sure you did but just in case you made sure that FILES_VOLUME_PATH is set right?

**Comment by ablwr** [comment]
AFAIK I haven't updated Docker and it hasn't set itself to autoupdate, nor has my Mac (maybe). I set FILES_VOLUME_PATH in a few ways... I tried without, with the .env, and with calling it and `&& ./quickstart.sh`

**Comment by yayoiukai** [comment]
Can you update Docker and try it again?

**Comment by ablwr** [comment]
I'm just going to use the test instance for now and hope this improves in the future.
